external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -170,6 +187,83 @@ def input_variables(self) -> Sequence:
             self.transform_factory.backward_names(set(self._model.input_variables))
         )
 
+    @property
+    def model_variables(self) -> Set[str]:
+        return self.transform_factory.backward_names(
+            set(self._model.input_variables) | set(self._model.output_variables)
+        )
+
+    @property
+    def variables(self) -> Set[str]:
+        return self.model_variables
+
+    @classmethod
+    def get_default_model(
+        cls, input_variables, output_variables
+    ) -> "TransformedParameters":
+        """used for testing"""
+        return TransformedParameters(
+            model=MicrophysicsConfig(
+                input_variables=input_variables,
+                direct_out_variables=output_variables,
+                architecture=ArchitectureConfig("dense"),
+            ),
+            loss=CustomLoss(loss_variables=output_variables),
+            use_wandb=False,
+        )
```

Comment:
The class needs to be passed input/output variables, but those aren't top-level attributes, and the loss needs to know the loss variables separately since output_variables are decoupled from the loss. I'll rename to `init_testing`.